### PR TITLE
DCOS-40635: add tests for empty nodes page

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
## Testing

🤷‍♂️ I could not reproduce the bug, so I only wrote an integration test for now

## Dependencies

- [ ] Get @GeorgiSTodorov Feedback on how to reproduce

### CODE REVIEW

File: plugins/nodes/src/js/components/NodesGridView.js [hunk 1]
```diff
@@ -121,18 +121,20 @@ var NodesGridView = React.createClass({
     );
   },
 
-  render() {
+  shouldRenderLoadingScreen() {
```
Comment (nLight) [inline plugins/nodes/src/js/components/NodesGridView.js]: ❤️

**Comment by nLight** [comment]
I think we good to go here. Putting a fix and a test in place when it's hard to repro an issue should be the way to go.
